iterators_task_1/student/src/impl.cpp

```diff
@@ -1 +1,22 @@
 #include "impl.h"
+
+#include <vector>
+#include <algorithm>
+#include <iostream>
+
+std::string reverseWords(const std::string& input)
+{
+    using ii_string = std::istream_iterator<std::string>;
+    using oi_string = std::ostream_iterator<std::string>;
+
+    std::istringstream iss(input);
```

Comment:
_please follow the same style of initialization ({})_

Reply:
Done